apps/fabric-website/src/pages/GetStarted/GetStartedPage.tsx

```diff
@@ -173,30 +173,35 @@ export class GetStartedPage extends React.Component<any, any> {
 
         <div className={styles.instructionsSection}>
           <h2 id="react">Get started with Fabric React</h2>
-          <p>
-            Use NPM to get Fabric components and core styling. All you need is{' '}
-            <a className={styles.getStartedLink} href="https://nodejs.org/en/">
-              node.js
-            </a>{' '}
-            and{' '}
-            <a className={styles.getStartedLink} href="http://gulpjs.com/">
-              gulp
-            </a>
-            .
-          </p>
+          <p>All you need to get started with Fabric is Node.js and npm.</p>
 
           <ol className={styles.steps}>
             <li>
-              <p>To install the Fabric React NPM package, from the root of your project, run:</p>
+              <p>
+                Install Node.js (at least 8.x) from the <a href="https://nodejs.org">Node.js website</a>.
+              </p>
+            </li>
+            <li>
+              <p>
+                Use npm to create a React app that uses TypeScript. Consider using Facebook's{' '}
```

Comment:
Just curious, why the explicit `{' '}`? I thought it might be for a line break. But judging from the screenshots above that doesn't seem like it.

Reply:
Per David's comment above, the space between "Facebook's" and "Create..." gets clipped at runtime, causing the words to run together. Here's what it looks like when I remove that:
![image](https://user-images.githubusercontent.com/307118/47872828-dddb3e00-ddcc-11e8-927a-b8469597cfc0.png)